wsf_scraping/spiders/msf_spider.py

```diff
@@ -0,0 +1,69 @@
+import scrapy
+from .base_spider import BaseSpider
+from wsf_scraping.items import MSFArticle
+
+
+class MSFSpider(BaseSpider):
+    name = 'msf'
+
+    custom_settings = {
+        'JOBDIR': 'crawls/msf'
+    }
+
+    def start_requests(self):
+        """Set up the initial request to the website to scrape."""
+
+        urls = [
+            'https://www.msf.org.uk/activity-reports',
+            'https://www.msf.org.uk/reports',
+        ]
```

Comment:
similarly here it seems that there could be a base class that runs by receiving urls which is the thing you change.